relentless/potential/potential.py

```diff
@@ -598,6 +598,7 @@ def energy(self, pair, potentials):
 
         """
         u = np.zeros_like(self.r)
+        u = u.astype(dtype=np.float64)
```

Comment:
This will result in a double-copy of `r`, so we should do the longer form:
```suggestion
        u = np.zeros(self.r.shape, dtype=np.float64)
```